docs/DeveloperGuide.md

```diff
@@ -818,13 +818,12 @@ testers are expected to do more *exploratory* testing. Each test case is to be e
 
 1. Initial launch
 
-    1. Test case: Download the jar file and copy into an empty folder. Double-click the jar file.<br>
-       Expected: Shows the GUI with a dashboard containing some sample data. The window size may not be optimum.
+    1. Test case: Download the `.jar` file and copy into an empty folder. Start *Productiv* from the terminal.<br>
```

Comment:
- I think don't need `` coz it is not code. 
- Can append something like 'i.e. type java -jar Productiv.jar' to 'start productive from the terminal' maybe?